ipet/parsing/ReaderManager.py

```diff
@@ -296,46 +302,71 @@ def collectData(self):
         """
         assert(self.testrun != None)
 
-        # sort the files by context
-        self.filestrings = sorted(self.filestrings, key = lambda x:self.sortingKeyContext(self.filenameGetContext(x)))
+        # sort the files by context, for example: outfiles should be read before solufiles
+        if not self.testrun.inputfromstdin:
```

Comment:
What about all the magic with reading settings and parameters from filenames then? 
Is that a problem if we have a mix of .out- and .solu-files?